application/app/components/Map/layers/thresholdsToColorsParams.ts

```diff
@@ -0,0 +1,12 @@
+import { ColorSpecification, ExpressionInputType, ExpressionSpecification } from 'maplibre-gl';
+
+import { Thresholds } from '../constants';
+
+export default function thresholdsToStepColorsParams(thresholds: Thresholds) {
+	const thresholdsEntries = Object.entries<ColorSpecification>(thresholds);
+
+	return thresholdsEntries.flatMap(([threshold, color]) => [+threshold, color]).slice(1) as [
```

Comment:
Je suis plus partant d'utiliser Number plutôt qu'un plus qui n'est pas très lisible.
```suggestion
	return thresholdsEntries.flatMap(([threshold, color]) => [Number(threshold), color]).slice(1) as [
```